spicy/runtime/src/tests/zlib.cc

```diff
@@ -28,6 +28,12 @@ TEST_CASE("decompress") {
             CHECK_EQ(zlib::finish(stream), ""_b);
         }
 
+        SUBCASE("raw deflate") {
+            stream = zlib::Stream(-15);
+            CHECK_EQ(zlib::decompress(stream, "\x33\x34\x84\x01\x2e\x00"_b), "1111111111\n"_b);
+            CHECK_EQ(zlib::finish(stream), ""_b);
```

Comment:
This mutates `stream` from the shared fixture. This isn't a bug, but still potentially confusing. I'd suggest we use a dedicated stream instead, e.g.,

```suggestion
            auto raw_stream = zlib::Stream(-15);
            CHECK_EQ(zlib::decompress(raw_stream, "\x33\x34\x84\x01\x2e\x00"_b), "1111111111\n"_b);
            CHECK_EQ(zlib::finish(raw_stream), ""_b);
```